decidim-dev/lib/decidim/dev/test/rspec_support/translation_helpers.rb

```diff
@@ -15,8 +15,11 @@ def translated(field, locale: I18n.locale)
   #
   # field - the field that holds the translations
   # locale - the ID of the locale to check
-  def have_i18n_content(field, locale: I18n.locale)
-    have_content(stripped(translated(field, locale: locale)))
+  # upcase - a boolean to indicate wether the string must be checked upcased or not.
```

Comment:
![](https://i.imgur.com/y8l9Sex.png)

You might want to change this to "whether"...